.gitignore

```diff
@@ -37,3 +37,6 @@ compat/src/test/rust/scenarios/target/
 compat/src/test/rust/scenarios/Cargo.lock
 compat/src/test/python/__pycache__/
 venv/
+compat/src/test/csharp/bin/
+compat/src/test/csharp/obj/
+/compat/src/test/csharp/Folder.DotSettings.user
```

Comment:
is this correct to have /? looks like an outlier

Reply:
yes, it works. the leading slash means a file matches iff it's in the exact path specified. 
while without the leading slash it would match also in `foo/compat/src/test/csharp/Folder.DotSettings.user` or `foo/bar/whatever/compat/src/test/csharp/Folder.DotSettings.user`